pom.xml

```diff
@@ -11,6 +11,12 @@
       <slf4j.version>1.7.7</slf4j.version>
     </properties>
 
+    <scm>
+      <connection>scm:git:git://github.com/SAP/devops-cm-client.git</connection>
+      <developerConnection>scm:git:ssh://github.com:SAP/devops-cm-client.git</developerConnection>
```

Comment:
Why do we need developer connection?

Reply:
The section here is a prerequisite documented [here](http://central.sonatype.org/pages/requirements.html#scm-information)